vtex-apps/wishlist
# Removed button `add item to cart` and adjusted screen scroll in wish list.

#### What is the purpose of this pull request?

As the title says.

 #### What problem is this solving?

In product details' screen, when add product to wish list and trigger `SEE LISTS` into toast, the lists are shown, but the button `ADD ITEM TO CART` stays are shown. Besides, the screen isn't scroll.

 #### How should this be manually tested?

1. Open a product;
2. In the product details, press the heart to add the product in the wish list;
3. Will be displayed a section with the wish lists, selected one and press `APPLY`;
4. Will be displayed a toast in wich to press the `SEE LISTS`;
5. **Observes that will be shown the wish lists, but the button `ADD ITEM TO CART` continues in the screen**;
6. Adds many lists until the lists exceed the screen max height. 
7. **Observes that the screen don't scroll**.

 #### Types of changes
